src/xenia/ui/window.h

```diff
@@ -53,6 +53,8 @@ class Window {
     return true;
   }
 
+  virtual bool set_icon_from_buffer(void *buffer, size_t size) = 0;
```

Comment:
This should be named SetIconFromBuffer. Only use lowercase underscored names for functions that directly return a variable.